internal/bminventory/inventory.go

```diff
@@ -1510,6 +1511,62 @@ func (b *bareMetalInventory) GetClusterDefaultConfig(_ context.Context, _ instal
 	return installer.NewGetClusterDefaultConfigOK().WithPayload(&body)
 }
 
+func setPlatformFromVendor(vendor models.SystemVendor, hostsPlatform *[]models.PlatformType) {
+	var supportedManufacturers = map[string]models.PlatformType{
+		"VMware, Inc.": models.PlatformTypeVsphere,
```

Comment:
hmmm @eliorerz are u sure that's only it? Did u check elastic data? I think I saw other strings as well.

Reply:
Also, lets have those strings in a more central place

Reply:
> hmmm @eliorerz are u sure that's only it? Did u check elastic data? I think I saw other strings as well.

@romfreiman I don't think that there are different manufacturers values - see https://kb.vmware.com/s/article/1005436